staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2367,6 +2394,11 @@ type Container struct {
 	// More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
 	// +optional
 	Resources ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,8,opt,name=resources"`
+	// Resources resize policy for the container.
```

Comment:
Is this field mutable or immutable?

Reply:
No, resize policy is not mutable as of now. IIRC, the policy is determined based on application characteristics and is unlikely to change from one call to the next.